sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -1895,8 +1959,16 @@
     .setDeleteHistoricalData(true)
     .create();
 
+  /**
+   * @deprecated since 5.2 No more design features
+   */
+  @Deprecated
   public static final String FILE_FEEDBACK_EDGES_KEY = "file_feedback_edges";
-  public static final Metric<Integer> FILE_FEEDBACK_EDGES = new Metric.Builder(FILE_FEEDBACK_EDGES_KEY, "Suspect file dependencies", Metric.ValueType.INT)
+  /**
+   * @deprecated since 5.2 No more design features
+   */
+  @Deprecated
+  public static final transient Metric<Integer> FILE_FEEDBACK_EDGES = new Metric.Builder(FILE_FEEDBACK_EDGES_KEY, "Suspect file dependencies", Metric.ValueType.INT)
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Remove the "transient" modifier from this field.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2065)
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)